### PR TITLE
The enhancement Requested, the cancellation by the Booking Id

### PR DESCRIPTION
**Pull request by al-amine**

### CODE REVIEW

**Review by kingjon3377 — APPROVED** [review]
The same complaints I had about the code I somewhat-grudgingly approved in #6 also apply here, but just like that this *does* meet the minimal quality standard, and just like that we need this to build the user-facing APIs.

**Comment by kingjon3377** [comment]
(For the record, this fixes #7.)

File: src/main/java/com/st/utopia/cancellationservice/model/Ticket.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	private Integer price;
 
 	private LocalDateTime reservationTimeout;
-
+
```
Comment (STOzaki) [inline src/main/java/com/st/utopia/cancellationservice/model/Ticket.java:47]: Why are you adding a tab here?

**Review by STOzaki — APPROVED** [review]
Look good. This is a side note. I would recommend working from a different branch than master. One, you can continue working on other things and not having to worry about having to add something new to the old branch because they are completely separate. Does that make sense?

**Pull request merged**